assets/js/components/ClusterDetails/ChecksSelection.jsx

```diff
@@ -102,6 +105,20 @@ export const ChecksSelection = ({ clusterId, cluster }) => {
     setLocalSavingSuccess(null);
   }, [loading, selectedChecks]);
 
+  if (catalogErrorCode == 'not_found') {
+    return (
+      <NotificationBox
+        icon={<EOS_ERROR className="m-auto" color="red" size="xl" />}
+        text={
+          <ReactMarkdown
+            className="markdown"
+            remarkPlugins={[remarkGfm]}
+          >{`Provider \`${cluster?.provider}\` does not support checks execution`}</ReactMarkdown>
+        }
+      />
+    );
```

Comment:
Why do we need ReactMarkdown if here we don't have any markdown to render?

Reply:
Good point! 
I believe Markdown has been used to render a backticked provider like
`gcp`
`aws`

Yes, sounds a bit overkill to use markdown here.
We could make another ui choice and just style the provider in a simpler way.

bold, red, whatever :smile:

Reply:
<b>text</b>
<strong>text</strong>
<p className="font-semibold-or-something-in-tailwind-doc">text</p>

:slightly_smiling_face:

Reply:
Thanks @dottorblaster and @nelsonkopliku . I will revisit this. @rtorrero is a big fan of codeblock, that's why we finally used that, but I can retry simply with bold hehe

Reply:
@arbulu89 no problem, you can either retry with bold or have a switch case that basically maps the constant `gcp`, `aws` and so on to plain cloud provider names. No need to display the raw constant!

Reply:
@dottorblaster The issue with bold is that I need to escape the `<b>` tags, otherwise they are added as simple text.
For that, I would need to import the npm `html-react-parser`. Is it alright?

Reply:
I don't think so, shall we merge this so we can iterate in a second moment?

Reply:
It works as it is. The styling is a minor thing in my opinion at the moment. The code blocs is pretty visible. And we can easily iterate later on